Sources/PackageGraph/Pubgrub.swift

```diff
@@ -573,6 +643,19 @@ public final class PubgrubDependencyResolver<
         // TODO: Handle pins
         assert(self.root != nil)
 
+        // Handle root, e.g. add dependencies and root decision.
+        solution.decide(Term(self.root!, .versionSet(.exact("1.0.0"))))
```

Comment:
You can extract root using guard to avoid force-unwrapping here and at line 653.